### PR TITLE
Backend refactoring

### PR DESCRIPTION
**Pull request by Gronner**
Closes #67 

Backend refactoring to achieve higher code quality following clean code

### CODE REVIEW

File: backend/models/entry.py [hunk 1]
```diff
@@ -24,3 +29,27 @@ def set_state(self, new_state):
...
+    entry_id = generate_entry_id(entry_name)
+    largest_previous_entry = db.session.query(Entry) \
+                                       .filter_by(queue=queue) \
+                                       .order_by(desc(Entry.ticket_number)) \
+                                       .limit(1) \
+                                       .first()
+    if largest_previous_entry is None:
+        ticket_number = 1
+    else:
+        ticket_number =largest_previous_entry.ticket_number + 1
```
Comment (ohaz) [inline backend/models/entry.py]: missing space

File: backend/utils/handle_get_queries.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from flask import abort
+def get_entry_detail_level(request):
+    """
+    Returns wether the detail level of an entry should be short or full.
+    """
+    entry_level_detail  = request.args.get('personDetails', None)
+    if entry_level_detail  not in ['short', 'full']:
```
Comment (ohaz) [inline backend/utils/handle_get_queries.py]: 2 spaces between `detail` and `not`

File: backend/models/entry.py (hunk 1 above)
Comment (Gronner) [reply backend/models/entry.py]: Fixed

File: backend/utils/handle_get_queries.py (hunk 2 above)
Comment (Gronner) [reply backend/utils/handle_get_queries.py]: Fixed

**Comment by Gronner** [comment]
Approved via Teams

**Pull request merged**